dashboard/app/views/sessions/lockout.html.haml

```diff
@@ -0,0 +1,4 @@
+#lockout-container{:"data-request-date" => (DateTime.now - 6).iso8601, :"data-delete-date" => (DateTime.now + 2).iso8601, :"data-pending-email" => "", :"data-api-url" => "/permissions"}
```

Comment:
```suggestion
#lockout-container{data: { request_date: 6.days.ago.iso8601, delete_date: 2.days.since.iso8601, pending_email: '', api_url: '/permissions'}}
```